lib/configs/recommended.js

```diff
@@ -0,0 +1,104 @@
+module.exports = {
+  extends: ['eslint:recommended', 'plugin:compat/recommended'],
+
+  plugins: ['compat', '@salesforce/eslint-plugin-aura'],
+
+  env: {
+    browser: true,
+  },
+
+  parserOptions: {
+    ecmaVersion: 5,
+  },
+
+  globals: {
+    $A: true,
+    AuraContext: true,
+    AuraSerializationService: true,
+    AuraExpressionService: true,
+    AuraEventService: true,
+    AuraLocalizationService: true,
+    AuraStorageService: true,
+    AuraStyleService: true,
+    MetricsService: true,
+    AuraDevToolService: true,
+    Component: true,
+    CKEDITOR: true,
+    FORCE: true,
+    moment: true,
+    exports: true,
+    iScroll: true,
+    Promise: true,
+  },
+
+  rules: {
+    // Best practices
+    // https://eslint.org/docs/rules/#best-practices
+    'array-callback-return': 'error',
+    'consistent-return': 'error',
+    'default-case': 'error',
+    'dot-notation': ['error', { allowKeywords: true }],
+    eqeqeq: ['error', 'smart'],
+    'guard-for-in': 'error',
+    'no-alert': 'error',
+    'no-caller': 'error',
+    'no-else-return': 'error',
+    'no-empty-function': [
+      'error',
+      {
+        allow: ['arrowFunctions', 'functions', 'methods'],
+      },
+    ],
+    'no-eval': 'error',
+    'no-extend-native': 'error',
+    'no-extra-bind': 'error',
+    'no-floating-decimal': 'error',
+    'no-implied-eval': 'error',
+    'no-iterator': 'error',
+    'no-labels': 'error',
+    'no-loop-func': 'error',
+    'no-multi-str': 'error',
+    'no-new': 'error',
+    'no-new-func': 'error',
+    'no-new-object': 'error',
+    'no-new-wrappers': 'error',
+    'no-octal-escape': 'error',
+    'no-proto': 'error',
+    'no-return-assign': 'error',
+    'no-return-await': 'error',
+    'no-script-url': 'error',
+    'no-self-compare': 'error',
+    'no-sequences': 'error',
+    'no-throw-literal': 'error',
+    'no-useless-concat': 'error',
+    'no-useless-escape': 'error',
+    'no-useless-return': 'error',
+    'no-unused-expressions': 'error',
```

Comment:
`no-unused-expressions` and `func-names` are both turned off by default in the [sfdx template](https://github.com/forcedotcom/salesforcedx-templates/pull/186/files), according to [this issue](https://github.com/forcedotcom/eslint-plugin-aura/issues/8) because they create too much noise with Aura. 

My opinion is that standard Aura code that follows best practices should not result in a linting error. We can do it as part of a separate PR, but we should run the recommended ruleset against some publicly documented code and disable any rules that error.